pkg/network/additional_networks.go

```diff
@@ -28,9 +30,15 @@ func renderRawCNIConfig(conf *operv1.AdditionalNetworkDefinition, manifestDir st
 	var err error
 	objs := []*uns.Unstructured{}
 
+	namespace := conf.Namespace
```

Comment:
I added a "getOr" function to the templating language, You can do something like this
```
namespace: {{getOr . "AdditionalNetworkNamespace" "default" }}
```

Up to you which you'd rather do.

Reply:
Updated.